### PR TITLE
#85: Add missing hacks.css file

### LINKED ISSUE #83: Add generated docs dir to gitignore
### TODO:
- Add generated docs dir to gitignore in order to avoid non intentional pushing generated docs to the main branch

### PR DESCRIPTION
**Pull request by marcinwrobel1986**
### THIS PR:
- added missing `hacks.css`
- added `html_static_path` directory to configuration

closes #83

### CODE REVIEW

File: docs/source/hacks.css [hunk 1]
```diff
@@ -0,0 +1,164 @@
...
...v-21.8c-19.6 1.6-19.7-14.6-31.6-23-18.7 20.6-31.6 40.8-58.9 51.1-12.7 4.8-19.6 10-25.9 21.8 34.9-16.4 91.2-13.5 98.8-10zM555.5 0l-.6 1.1-.3.9.6-.6zm-59.2 382.1c-33.9-56.9-75.3-118.4-150-115.5l-.3-6c-1.1-13.5 32.8 3.2 35.1-31l-14.4 7.2c-19.8-45.7-8.6-54.3-65.5-54.3-14.7 0-26.7 1.7-41.4 4.6 2.9 18.6 2.2 36.7-10.9 50.3l19.5 5.5c-1.7 3.2-2.9 6.3-2.9 9.8 0 21 42.8 2.9 42.8 33.6 0 18.4-36.8 60.1-54.9 60.1-8 0-53.7-50-53.4-60.1l.3-4.6 52.3-11.5c13-2.6 12.3-22.7-2.9-22.7-3.7 0-43.1 9.2-49.4 10.6-2-5.2-7.5-14.1-13.8-14.1-3.2 0-6.3 3.2-9.5 4-9.2 2.6-31 2.9-21.5 20.1L15.9 298.5c-5.5 1.1-8.9 6.3-8.9 11.8 0 6 5.5 10.9 11.5 10.9 8 0 131.3-28.4 147.4-32.2 2.6 3.2 4.6 6.3 7.8 8.6 20.1 14.4 59.8 85.9 76.4 85.9 24.1 0 58-22.4 71.3-41.9 3.2-4.3 6.9-7.5 12.4-6.9.6 13.8-31.6 34.2-33 43.7-1.4 10.2-1 35.2-.3 41.1 26.7 8.1 52-3.6 77.9-2.9 4.3-21 10.6-41.9 9.8-63.5l-.3-9.5c-1.4-34.2-10.9-38.5-34.8-58.6-1.1-1.1-2.6-2.6-3.7-4 2.2-1.4 1.1-1 4.6-1.7 88.5 0 56.3 183.6 111.5 229.9 33.1-15 72.5-27.9 103.5-47.2-29-25.6-52.6-45.7-72.7-79.9zm-196.2 46.1v27.2l11.8-3.4-2.9-23.8zm-68.7-150.4l24.1 61.2 21-13.8-31.3-50.9zm84.4 154.9l2 12.4c9-1.5 58.4-6.6 58.4-14.1 0-1.4-.6-3.2-.9-4.6-26.8 0-36.9 3.8-59.5 6.3z"/></svg>');
```
Comment (fnrizzi) [inline docs/source/hacks.css:151]: can you please remove all this? I forgot to delte it since we don't use it

File: docs/source/hacks.css [hunk 2]
```diff
@@ -0,0 +1,164 @@
...
+  border-color: rgb(43, 155, 70);
+}
+.admonition.pied-piper > .admonition-title {
+  background-color: rgba(43, 155, 70, 0.1);
+  border-color: rgb(43, 155, 70);
+}
+.admonition.pied-piper > .admonition-title::before {
+  background-color: rgb(43, 155, 70);
+  -webkit-mask-image: var(--icon--pied-piper);
+  mask-image: var(--icon--pied-piper);
```
Comment (fnrizzi) [inline docs/source/hacks.css:163]: same for all these pied piper stuff

File: docs/source/hacks.css (hunk 1 above)
Comment (marcinwrobel1986) [reply docs/source/hacks.css:151]: ok, sec

**Comment by fnrizzi** [comment]
looks good! thanks

**Review by fnrizzi — APPROVED** [review]

**Pull request merged**